yubico-piv-tool.json

```diff
@@ -0,0 +1,33 @@
+{
+    "homepage": "https://developers.yubico.com/yubico-piv-tool/",
+    "version": "1.5.0",
+    "license": "BSD 2-Clause",
```

Comment:
Needs a dash

Reply:
Thank you. But I can't understand what you say.

Reply:
Sorry, `"BSD 2-Clause"` should be `"BSD-2-Clause"`